core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -73,6 +73,18 @@ public static long getLong(String property, JsonNode node) {
     return pNode.asLong();
   }
 
+  public static Long getLongOrNull(String property, JsonNode node) {
+    if (!node.has(property)) {
+      return null;
+    }
+
+    Preconditions.checkArgument(node.has(property), "Cannot parse missing long %s", property);
```

Comment:
No point anymore to this check anymore , right ?

Reply:
correct, updated